### ISSUE TITLE
Question: Is `this.ackRequests.has(requestId)` a redundant condition?

### ISSUE THREAD

**Issue by zishang520** [socketio/socket.io-redis-adapter#549]
If `requestId` does not exist in `this.requests`, but exists in `this.ackRequests`: 

https://github.com/socketio/socket.io-redis-adapter/blob/cdb55353f83c78cabe9788683e4dd93ac4cd50c9/lib/index.ts#L529

L537 will have an exception:

https://github.com/socketio/socket.io-redis-adapter/blob/cdb55353f83c78cabe9788683e4dd93ac4cd50c9/lib/index.ts#L537

**Comment by darrachequesne** [comment]
Good catch! It looks like it is indeed redundant. Would you mind opening a PR?

**Comment by zishang520** [comment]
PR submitted
